src/components/NewClusterModal.js

```diff
@@ -227,18 +290,20 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
           disabled: isSelectedImageInputted && isCustomImageInvalid,
           tooltip: isSelectedImageInputted && isCustomImageInvalid && 'Enter a valid docker image to use',
           onClick: () => {
-            if (isSelectedImageInputted) {
+            if (isSelectedImageInputted && !this.canUpdate()) {
```

Comment:
if the user has already seen the warning, and already has a cluster with this image, we don't want to warn them again and proceed with the replace logic